src/service/mod.rs

```diff
@@ -45,5 +45,5 @@ pub(crate) use self::http::HttpService;
 pub(crate) use self::make::{MakeConnection, MakeServiceRef};
 pub(crate) use self::oneshot::{oneshot, Oneshot};
 
-pub use self::make::make_service_fn;
-pub use self::util::service_fn;
+pub use self::make::{make_service_fn, MakeServiceFn};
```

Comment:
I'd prefer to keep them un-exported for the time being.

Reply:
Done (though I find un-exported types like this makes unnecessarily hard to understand what traits they implement and how they implement them).